### ISSUE TITLE
add logging

### ISSUE THREAD

**Issue by davidlfox** [davidlfox/DrugBot#6]
probably use azure table storage unless theres a quick, locally testable way to use ELK stack

**Comment by davidlfox** [comment]
add some kpi's e.g. superlatives. most money owed to loan shark, favorite drug to buy/sell, favorite location, etc

**Comment by davidlfox** [comment]
fixed under 6ad7898057cae9dad57e84f2d29e8b5d2d2fbdb4 with the ability to add custom logging e.g.
```
var ai = new TelemetryClient();
var props = new Dictionary<string, string>();
props.Add("messagetext", activity.Text);
ai.TrackException(ex, props);
```
